saskatoon/saskatoon/settings.py

```diff
@@ -21,7 +21,7 @@
 BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
 
 # Load the environment variables from .env file.
-dotenv = os.path.join(BASE_DIR, '.env')
+dotenv = os.path.join(BASE_DIR, 'saskatoon/.env')
```

Comment:
BASE_DIR should be `saskatoon` here, please double check

Reply:
i started the development server with python3 saskatoon/manage.py runserver 8000 as is the install.md. A nd i got this error
![env](https://user-images.githubusercontent.com/78615753/162662837-35f8776c-a785-4ac7-9bc9-80eaecf4b6ec.png)

Reply:
so i manually had to set 'saskatoon/.env' which fixes the problem. I guess it ought to be in the saskatoon folder not the saskatoon/saskatoon folder.

Reply:
where did you place your `.env` file exactly? If you place it at the same location than the template file it should work since:
- `BASE_DIR` should give `../saskatoon/`
- `dotenv` should be `../saskatoon/.env` as indicated in the INSTALL.md, not `saskatoon/saskatton/.env`